api/api/utils/image_proxy/__init__.py

```diff
@@ -59,24 +76,23 @@ def get_request_params_for_extension(
 
 
 def get(
-    image_url: str,
-    media_identifier: str,
-    media_provider: str,
-    accept_header: str = "image/*",
-    is_full_size: bool = False,
-    is_compressed: bool = True,
+    media_info: ImageProxyMediaInfo,
+    proxy_config: ImageProxyConfig = ImageProxyConfig(),
```

Comment:
This might be a bad name, maybe "image proxy request config"?

Reply:
Yea I think `request_proxy_config` for the variable itself might be better 🤔